spec/cli_spec.rb

```diff
@@ -9,14 +9,22 @@
 
 RSpec.describe 'CLI' do
   it 'renders reproducible dumps' do
-    system('bundle', 'exec', 'rails-response-dumper', chdir: APP_DIR, exception: true)
+    env = { 'TMPDIR' => tmpdir, 'FILENAME' => 'test.out' }
+    cmd = %w[bundle exec rails-response-dumper]
+    stdout, stderr, status = Open3.capture3(env, *cmd, chdir: APP_DIR)
+
+    expect(stdout).to eq("...\n")
+    expect(stderr).to be_empty
```

Comment:
`be_empty` is imprecise about the type.

```suggestion
    expect(stderr).to eq('')
```